string tech
string tech2
string mcc
string mnc
string cellid

